--- .spacemacs	2019-10-30 18:19:45.140866120 -0400
+++ .local/src/dotfiles/src/spacemacs	2019-10-30 17:58:13.279537372 -0400
@@ -37,6 +37,7 @@
      ;; <M-m f e R> (Emacs style) to install them.
      ;; ----------------------------------------------------------------
      helm
+     drbr
      ;; auto-completion
      ;; better-defaults
      emacs-lisp
@@ -118,7 +119,10 @@
    ;; List sizes may be nil, in which case
    ;; `spacemacs-buffer-startup-lists-length' takes effect.
    dotspacemacs-startup-lists '((recents . 5)
-                                (projects . 7))
+                                (projects . 7)
+                                (agenda . 5)
+                                (todos . 5)
+                                (bookmarks 5))
    ;; True if the home buffer should respond to resize events.
    dotspacemacs-startup-buffer-responsive t
    ;; Default major mode of the scratch buffer (default `text-mode')
@@ -262,7 +266,7 @@
    dotspacemacs-line-numbers nil
    ;; Code folding method. Possible values are `evil' and `origami'.
    ;; (default 'evil)
-   dotspacemacs-folding-method 'evil
+   dotspacemacs-folding-method 'origami
    ;; If non-nil smartparens-strict-mode will be enabled in programming modes.
    ;; (default nil)
    dotspacemacs-smartparens-strict-mode nil

